config/webpack.config.js

```diff
@@ -186,6 +186,7 @@ const configGenerator = (options) => {
       new webpack.DefinePlugin({
         __BUILDTYPE__: JSON.stringify(options.buildtype),
         __SAMPLE_ENABLED__: (process.env.SAMPLE_ENABLED === 'true'),
+        __VIC_RATE_LIMIT_ENABLED__: (JSON.stringify(options.buildtype) === 'development' || JSON.stringify(options.buildtype) === 'staging' || process.env.VIC_RATE_LIMIT_ENABLED === 'true'),
```

Comment:
I thought we wanted to rate-limit on dev and prod but _not_ on staging?

Reply:
yep 👍 👍 👍